src/components/CreditCard.jsx

```diff
@@ -0,0 +1,28 @@
+import React from 'react'
+
+function CreditCard(props) {
+    const {type, number, expirationMonth, expirationYear, bank, owner, bgColor, color} = props
+  
+  type = ["Visa", "Master card"]
```

Comment:
You are already getting `type` from the props, so you can't create a new variable with the same name. If you take out this line of code, the component will work.